couchdb_cluster_admin/add_replica_node.py

```diff
@@ -16,7 +18,7 @@ def _update_db_doc_with_new_node(node_details, db_name, new_node):
     db_doc = do_node_local_request(node_details, '_dbs/{}'.format(db_name))
     by_node = db_doc['by_node']
     if new_node in by_node:
-        print('  Node "{}" already has shards for db "{}"'.format(new_node, db_name))
+        print(('  Node "{}" already has shards for db "{}"'.format(new_node, db_name)))
```

Comment:
these print statements are already compliant so the extra braces are incorrect

Reply:
Yep, they are. I guess the script regexed for print and added these extra braces and I just realized you can add infinite braces in the print and it will work fine 😅 

Btw, I have fixed them.